tensorflow/core/kernels/map_kernels.h

```diff
@@ -106,37 +106,37 @@ class TensorMapSize : public OpKernel {
   }
 };
 
-class TensorMapInsert : public OpKernel {
```

Comment:
Are these just swapped? If there is no strong reason to swap maybe keep them as it is to keep the PR simple. Same for the op registrations.

Reply:
Alternatively, you can update the PR description to be more general, maybe something like "updates to tensor map ops: <update 1>, <update 2>".

Reply:
+1 to Dan's comment if there are actual changes here. It was just a little hard to tell from the diff.

Reply:
Yup, just wanted to make the order of ops consistent between kernels and registrations.